src/models/messages.js

```diff
@@ -14,15 +14,16 @@ function * sendMessage (message, requestingUser) {
     throw new InvalidBodyError(error, validationResult.errors)
   }
 
-  const mode1 = !!message.account
-  const mode2 = !!(message.from && message.to)
-  if ((mode1 && mode2) || (!mode1 && !mode2)) {
-    throw new InvalidBodyError('Expected either message.account OR message.from/to')
+  // For backwards compatibility.
+  if (message.account) {
+    message.to = message.account
```

Comment:
You should still check (and throw an exception) if `to` or `from` are already (also) set.